commons-annotations/src/main/scala/com/avsystem/commons/rpc/RPCName.scala

```diff
@@ -1,9 +0,0 @@
-package com.avsystem.commons
-package rpc
-
-import scala.annotation.StaticAnnotation
-
-/**
-  * You can use this annotation on overloaded RPC methods to give them unique identifiers for RPC serialization.
-  */
-class RPCName(val name: String) extends StaticAnnotation
```

Comment:
Can we deprecate it first and remove in the next release?

Reply:
I actually renamed it to `rpcName` :trollface: to retain some naming convention. To be honest, I'd like to have a single annotation for both `GenCodec` (which currently uses `@name`) and RPC. So I will probably introduce some deprecated type aliases (although I'm not sure if they work with annotations).

Reply:
I just want to get deprecation warning (with migration instructions) instead of class not found error. :)

Reply:
I made an alias in [Udash PR](https://github.com/UdashFramework/udash-core/pull/187)